enclave-runner/src/usercalls/mod.rs

```diff
@@ -545,6 +552,43 @@ fn trap_attached_debugger(tcs: usize) {
     }
 }
 
+fn connect_stream_dflt(
+        addr: &[u8],
+        local_addr: Option<&mut OutputBuffer>,
+        peer_addr: Option<&mut OutputBuffer>,
+    ) -> IoResult<Box<SyncStream>> {
+        let addr = str::from_utf8(addr).map_err(|_| IoErrorKind::ConnectionRefused)?;
+        let stream = TcpStream::connect(addr)?;
+        if let Some(local_addr) = local_addr {
+            match stream.local_addr() {
+                Ok(local) => local_addr.set(local.to_string().into_bytes()),
+                Err(_) => local_addr.set(&b"error"[..]),
+            }
+        }
+        if let Some(peer_addr) = peer_addr {
+            match stream.peer_addr() {
+                Ok(peer) => peer_addr.set(peer.to_string().into_bytes()),
+                Err(_) => peer_addr.set(&b"error"[..]),
+            }
+        }
+    Ok(Box::new(stream))
+}
+
+pub trait UsercallExtension : 'static + Send + Sync + std::fmt::Debug {
+    fn connect_stream(
+        &self,
+        _addr: &[u8],
+        _local_addr: Option<&mut String>,
+        _peer_addr: Option<&mut String>,
```

Comment:
Don't prefix with `_` as this will show in the docs. Either use `let _ =` statements in the function body, or silence the unused lint for this function.